fableous-fe/src/components/Navbar.tsx

```diff
@@ -39,14 +40,16 @@ export default function Navbar() {
               component={Link}
               to="/profile"
             >
-              Profile
+              <Icon fontSize="small">person</Icon>
```

Comment:
Move button icons to startIcon, like https://github.com/deco-finter/fableous/blob/master/fableous-fe/src/components/ClassroomItem.tsx#L195